stubs/redis/redis/cluster.pyi

```diff
@@ -106,47 +106,49 @@ class RedisCluster(AbstractRedisCluster, RedisClusterCommands[_StrType], Generic
     def close(self) -> None: ...
 
 class ClusterNode:
-    host: Any
-    port: Any
-    name: Any
-    server_type: Any
-    redis_connection: Any
-    def __init__(self, host, port, server_type: Any | None = ..., redis_connection: Any | None = ...) -> None: ...
-    def __eq__(self, obj): ...
+    host: str
+    port: int
+    name: str
+    server_type: str | None
+    redis_connection: Redis[Any] | None
+    def __init__(
+        self, host: str, port: int, server_type: str | None = ..., redis_connection: Redis[Any] | None = ...
+    ) -> None: ...
+    def __eq__(self, obj: object) -> bool: ...
     def __del__(self) -> None: ...
 
 class LoadBalancer:
-    primary_to_idx: Any
-    start_index: Any
+    primary_to_idx: dict[str, int]
+    start_index: int
     def __init__(self, start_index: int = ...) -> None: ...
-    def get_server_index(self, primary, list_size): ...
+    def get_server_index(self, primary: str, list_size: int) -> int: ...
     def reset(self) -> None: ...
 
 class NodesManager:
-    nodes_cache: Any
-    slots_cache: Any
-    startup_nodes: Any
-    default_node: Any
-    from_url: Any
-    connection_kwargs: Any
-    read_load_balancer: Any
+    nodes_cache: dict[str, ClusterNode]
+    slots_cache: dict[str, list[ClusterNode]]
+    startup_nodes: dict[str, ClusterNode]
+    default_node: ClusterNode | None
+    from_url: bool
+    connection_kwargs: dict[str, Any]
+    read_load_balancer: LoadBalancer
     def __init__(
         self,
         startup_nodes,
         from_url: bool = ...,
         require_full_coverage: bool = ...,
-        lock: Incomplete | None = ...,
+        lock: Lock | None = ...,
         dynamic_startup_nodes: bool = ...,
-        **kwargs,
+        **kwargs: Any,
     ) -> None: ...
-    def get_node(self, host: Any | None = ..., port: Any | None = ..., node_name: Any | None = ...): ...
-    def update_moved_exception(self, exception) -> None: ...
-    def get_node_from_slot(self, slot, read_from_replicas: bool = ..., server_type: Any | None = ...): ...
-    def get_nodes_by_server_type(self, server_type): ...
-    def populate_startup_nodes(self, nodes) -> None: ...
-    def check_slots_coverage(self, slots_cache): ...
-    def create_redis_connections(self, nodes) -> None: ...
-    def create_redis_node(self, host, port, **kwargs): ...
+    def get_node(self, host: str | None = ..., port: int | None = ..., node_name: str | None = ...) -> ClusterNode | None: ...
```

Comment:
Same as above

```suggestion
    def get_node(self, host: str | None = ..., port: int | str | None = ..., node_name: str | None = ...) -> ClusterNode | None: ...
```

Reply:
Here `post` can be `str`, because it is only used for string formatting (but, I personally think that it should not be used that away).